README.md

```diff
@@ -1,9 +1,9 @@
 redux
 =========================
 
-An experiment in fully hot-reloadable Flux.  
```

Comment:
Oh sorry, I have `"trim_trailing_white_space_on_save": true` on my sublime :)
Will try to revert it...